rosetta-source/src/main/resources/result-json-files/fpml-5-13/products/commodity-derivatives/com-ex41-oil-asian-barrier-option-strip.json

```diff
@@ -218,20 +233,32 @@
                   }
                 },
                 "meta" : {
-                  "globalKey" : "a12e5c18"
+                  "globalKey" : "6e98b82c"
                 }
               } ],
               "meta" : {
-                "globalKey" : "a12e5c18"
+                "globalKey" : "6e98b82c"
               }
             }
           },
+          "productIdentifier" : [ {
+            "identifier" : {
+              "value" : "OIL-BRENT-IPE",
+              "meta" : {
```

Comment:
This `productIdentifier` should not be mapped. I believe it's the reason the failed mappings increases, because ""OIL-BRENT-IPE" gets mapped 2x.

What is strange is that `productIdentifier` is not even an attribute of `ContractualProduct` (see comment above), so how can it be mapped??